controllers/apply/lib/joi-extensions/full-name.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = function fullName(joi) {
+    return {
+        name: 'fullName',
+        base: joi.object({
+            firstName: joi
+                .string()
+                .max(40)
```

Comment:
Out of interest, why the different maxlength between first/last name? Might as well make them the same for both (cf. https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/).

Reply:
(or is this a Salesforce thing?)

Reply:
This is a Salesforce limit sadly.

Reply:
I'm sad that my future child Llanfairpwllgwyngyllgogerychwyrndrobwllllantysiliogogogoch (named for my newly-discovered love of the Welsh language) will be unable to apply for funding.